src/containers/SalaryWorkTime/TimeAndSalaryBoard.js

```diff
@@ -1,32 +0,0 @@
-import { connect } from 'react-redux';
-import { bindActionCreators } from 'redux';
-import TimeAndSalaryBoard from '../../components/TimeAndSalary/TimeAndSalaryBoard';
```

Comment:
should `components/TimeAndSalary/TimeAndSalaryBoard` be removed ?

Reply:
Yes thank you!!